.evergreen.yml

```diff
@@ -341,14 +355,16 @@ functions:
     params:
       working_dir: parsley
       shell: bash
+      env: 
```

Comment:
Non-blocking suggestion: would it be better to use `subprocess.exec` instead of `shell.exec` throughout this config?

Reply:
for sure! I fixed one of the easy ones, but the other ones make me a bit nervous so I'll open a ticket to audit this usage!